src/resources/payments/PaymentsResource.ts

```diff
@@ -55,7 +55,12 @@ export default class PaymentsResource extends Resource<PaymentData, Payment> {
   public create(parameters: CreateParameters, callback: Callback<Payment>): void;
   public create(parameters: CreateParameters) {
     if (renege(this, this.create, ...arguments)) return;
-    return this.network.post(this.getResourceUrl(), parameters);
+    const { include, ...data } = parameters;
+    let query: { include: PaymentInclude[] | PaymentInclude } | undefined = undefined;
+    if (include != undefined) {
+      query = { include };
+    }
```

Comment:
I would write this a little bit different, because I feel that's more readable and makes nice use of type inference. Although I'll leave it up to you.
`const query = include ? { include } : undefined;`

Reply:
The reason I didn't, is because I don't want to silently drop the parameter if it is (accidentally) falsy.

I'll change it to this:
```javascript
const query = include != undefined ? { include } : undefined;
```
That's probably the cleanest solution.

Reply:
You need to have trust in the types 😄